new_builder_evaluation/app/infrastructure/repositories/model.py

```diff
@@ -16,7 +16,9 @@ def __init__(self) -> None:
 
     def update_light_model(self, id: int, light_model: str) -> dict:
         instance = self.session.query(self.model).filter(self.model.id == id).first()
+        light_model = f"{light_model}/model/single_evaluation"
         instance.light_model = light_model
+        instance.deployment_status = "deployed"
```

Comment:
This status can't be changed later, right? Models are deployed and the users have no option to stop the deployment?

Reply:
the status can be changed at any time. The idea is that already at this moment the model would be in production.